qemu/lib/iozone.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import os
+
+from virttest import utils_test
+from virttest import utils_misc
+from virttest import postprocess_iozone
+
+
+class IozoneTest(object):
+    """
+    Prepare iozone env, run iozone on guest, clean up iozone env.
+    """
+    def __init__(self, vm, params, test):
+        """
+        run setup of iozone test
+        """
+        self.params = params
+        self.session = vm.wait_for_login()
+        self.test = test
+        self.results_path = ""
+        self.timeout = float(params.get("iozone_timeout", 240))
+
+    def before_run_iozone(self):
+        """
+        Prepare for iozone test
+        """
+        raise NotImplementedError()
+
+    def run_iozone_in_guest(self, disk_index, mountpoint):
+        """
+        Run iozone test in guest
+
+        :param disk_index: index to find disk under test
+        :param mountpoint: mountpoint/disk letter assigned to disk under test
+        :return: iozone run status and result.
+        """
+        iozone_test_cmd_default = "./iozone -azR -r 64k -n 125M -g 512M -i 0 -i 1 %s"
+        iozone_test_cmd = self.params.get("iozone_test_cmd", iozone_test_cmd_default)
+        if self.params.get("os_type") == "windows":
+            if self.params.get("format_disk", "yes") == "yes":
```

Comment:
Suggest move disk format to before_run_iozone to separate disk format and iozone run.
Sometimes, we need to run iozone test 2 times at one case, when first iozone test, the disk has been formated(the "format_disk"=="yes" in cfg), when second iozone test, it will do disk format again, it no needed.

Reply:
As luckyh's suggestion, remove format_windows_disk from this class. Instead, we call it in our test cases when needed.